### ISSUE TITLE
on job error ensure exit code is not 0 and "status-message" contains the actual error message

### ISSUE THREAD

**Issue by konstan** [nuvla/job-engine#38]
Here is the case when deployment_start on docker_container failed by the job status was successful.
```
nuvla_job-executor.1.mte0ajx5lhy3@machineb588bc1f-a887-4701-a982-ca4c7f95c698    | 2019-05-29 07:01:33,509 - INFO - job_processor_Stan_1 - executor.py:51 - Got new job/315e78ab-43a8-471c-b141-1f320fed416c.
nuvla_job-executor.1.mte0ajx5lhy3@machineb588bc1f-a887-4701-a982-ca4c7f95c698    | 2019-05-29 07:01:33,543 - INFO - job_processor_Stan_1 - deployment_start.py:160 - Job started for deployment/c38f060a-2694-4bfc-86a2-00b7e33ece50.
nuvla_job-executor.1.mte0ajx5lhy3@machineb588bc1f-a887-4701-a982-ca4c7f95c698    | 2019-05-29 07:01:33,600 - INFO - job_processor_Stan_1 - docker_connector.py:78 - Connecting to endpoint https://test.nuvla.io:2376
nuvla_job-executor.1.mte0ajx5lhy3@machineb588bc1f-a887-4701-a982-ca4c7f95c698    | 2019-05-29 07:01:33,660 - ERROR - job_processor_Stan_1 - deployment_start.py:169 - Failed to start deployment deployment/c38f060a-2694-4bfc-86a2-00b7e33ece50: HTTPSConnectionPool(host='test.nuvla.io', port=2376): Max retries exceeded with url: /services/create (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_BAD_CERTIFICATE] sslv3 alert bad certificate (_ssl.c:1056)')))
nuvla_job-executor.1.mte0ajx5lhy3@machineb588bc1f-a887-4701-a982-ca4c7f95c698    | 2019-05-29 07:01:33,787 - INFO - job_processor_Stan_1 - job.py:136 - Great, job/315e78ab-43a8-471c-b141-1f320fed416c is now in final state; Removed from queue.
nuvla_job-executor.1.mte0ajx5lhy3@machineb588bc1f-a887-4701-a982-ca4c7f95c698    | 2019-05-29 07:01:33,787 - INFO - job_processor_Stan_1 - executor.py:72 - Successfully finished job/315e78ab-43a8-471c-b141-1f320fed416c.
```

**Comment by konstan** [comment]
`status-message` is part of `/job` schema.

**Comment by 0xbase12** [comment]
Fixed in commit efeca53167fea0a973c86f429c7bee7320f2e697
